packages/embark/src/lib/modules/codeRunner/index.js

```diff
@@ -16,8 +14,6 @@ class CodeRunner {
     this.ipc = options.ipc;
     this.vm = new VM({
       sandbox: {
-        IpfsApi,
-        Web3,
```

Comment:
Does that mean they are not available anymore?

Reply:
No, I register them inside their respective modules instead. IMO, it makes way more sense, since codeRunner shouldn't know about those.

Reply:
Agree :+1: